lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -297,6 +428,47 @@ public function render_language_dictionary( $args = array() ) {
 				__( 'List of all modifiable options', 'wp-auth0' )
 			)
 		);
+		$this->render_field_description(
+			__( 'NOTE: This field is deprecated and will be removed in the next major release. ', 'wp-auth0' ) .
+			__( 'Use a languageDictionary property the Extra Settings field below to change text.', 'wp-auth0' )
+		);
+	}
+
+	/**
```

Comment:
All of these methods existed in the plugin already (see deprecated methods above).